### PR TITLE
Torch and CUDA

### PR DESCRIPTION
**Pull request by hadim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

- Only build for CUDA >=11.2
- Build for torch 1.12 and 1.13

### CODE REVIEW

**Comment by hadim** [comment]
Should I modify the string number to add the torch version to it? This is more convenience here.

**Comment by hadim** [comment]
Closing here. See https://github.com/conda-forge/pyg-lib-feedstock/issues/2 for context.

**Comment by hadim** [comment]
So far the failing builds are related to network error. I will wait for all then to finish before relaunching.

**Comment by hadim** [comment]
LGTM here.

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pytorch:
+  - '1.12'
```
Comment (hmaarrfk) [inline recipe/conda_build_config.yaml:2]: Was it too hard to build 1.11? Didn't you need that version?

Comment (hadim) [reply recipe/conda_build_config.yaml:2]: ahah I am confused now. I understood it was too much burden for you to build for many torch versions.

As for my specific need I think >=1.12 should be fine.

Also I spotted some numerical errors due to 1.11 making the tests to fail and I am a bit lazy to look into that right now.

Given the above, I propose to skip 1.11 for now and I will rebuild for it in the future if I really need it.

Comment (h-vetinari) [reply recipe/conda_build_config.yaml:2]: Any reason you need to build for two pytorch versions? Can't we just let the migrators take care of this?

Comment (hadim) [reply recipe/conda_build_config.yaml:2]: It's not urgent for me. I was actually wondering that at https://github.com/conda-forge/pyg-lib-feedstock/issues/2#issuecomment-1371267833

Comment (hmaarrfk) [reply recipe/conda_build_config.yaml:2]: It is too much for me to builld old torch versions with new dependencies. Eventually, you won't be in a position to build new packages, with new pinnings, with the old pytorch packages.

But if it works, and you need refreshed packages for whatever reason, then this approach is fine so long as it continues to work.

**Comment by hadim** [comment]
@conda-forge-admin, please rerender

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (hadim) [reply recipe/conda_build_config.yaml]: Ok i will merge here then.

I removed the 1.13 builds and will wait for the bot to bump it.

**Pull request merged**